src/Okta.Sdk/Api/OAuthApi.cs

```diff
@@ -169,7 +169,7 @@ public async Task<ApiResponse<OAuthTokenResponse>> GetBearerTokenWithHttpInfoAsy
 
             _dpopProofJwtGenerator.RotateKeys();
             var jwtSecurityToken = _clientAssertionJwtGenerator.GenerateJwt();
-            var scopes = string.Join("+", Configuration.Scopes);
+            var scopes = string.Join(" ", Configuration.Scopes);
```

Comment:
Since this file is generated, we'll need to update the templates and re-generate the sdk using the openapi-generator.